### PR TITLE
Revise the type of HTTP Proxy mentioned

### PR DESCRIPTION
**Pull request by mjivraja**
Presently, in setting up the Project in the IPv6 network, we mention only the IPv4 type of HTTP Proxy server which is incorrect. Because, the HTTP Proxy server configured with IPv4 only cannot communicate with the Project configured in the IPv6 network, except CDN. Hence, it should support both, IPv4 and IPv6.

Please cherry-pick my commits into:

* [X] Foreman 3.8/Katello 4.10
* [X] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [X] Foreman 3.6/Katello 4.8
* [X] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6)
* [X] Foreman 3.4/Katello 4.6 (EL8 only)
* [X] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [X] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [X] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

File: guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc
Comment (maximiliankolb) [inline guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc:24]:
```suggestion
* You must configure {Project} to use this HTTP proxy server as default proxy supporting both IPv4 and IPv6.
```

**Review by maximiliankolb — COMMENTED** [review]
One small suggestion; change itself LGTM. Can someone else please ACK or do you have a link to another PR in foreman or bugzilla?

**Comment by mjivraja** [comment]
> One small suggestion; change itself LGTM. Can someone else please ACK or do you have a link to another PR in foreman or bugzilla?

@ekohl Can you share your view on this?

File: guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc
Comment (ekohl) [inline guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc:21]: Unrelated, but we should improve conditionals here so we only show the requirement if needed. Then drop this line.

I think line 14 can be changed to `ifndef::satellite`. While it mentions content, it is done in a way that is generic enough to also apply to non-Katello.

File: guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc
Comment (ekohl) [reply guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc:24]: I think this added part makes the sentence very long and hard to read. Perhaps this is easier?
```suggestion
* You must configure {Project} to use this dual stack (supporting both IPv4 and IPv6) HTTP proxy server as the default proxy.
```

**Review by ekohl — COMMENTED** [review]
ACK on the technical side: this should have always been a dual stack HTTP proxy requirement.

File: guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc
Comment (mjivraja) [reply guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc]: Addressed :+1:

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by maximiliankolb** [comment]
Merged to master :heavy_check_mark: 
I could not cherry-pick to 3.9 yet because someone has force pushed. See https://github.com/theforeman/foreman-documentation/pull/2644 

cc @mjivraja

**Comment by mjivraja** [comment]
> Merged to master ✔️ I could not cherry-pick to 3.9 yet because someone has force pushed. See #2644
> 
> cc @mjivraja

Thank you @maximiliankolb for tagging this PR in `2644`. Please let me know if we require separate PR for conflicts. I am assuming cherry-picking will begin for this PR once the `2644` is merged.

**Comment by maximiliankolb** [comment]
Cherry-picks: :heavy_check_mark: 
   24f50d249..584dfd1a6  3.9 -> 3.9
   0d87b850a..3afd09c65  3.8 -> 3.8
   f70658bc9..0bc5c2201  3.7 -> 3.7
   0e1cfa44c..244dbdcc1  3.6 -> 3.6
   eeafa284e..56da30caa  3.5 -> 3.5
   e3c9721f2..4cc580406  3.4 -> 3.4

There was a merge conflict on 3.3. If you want to backport to 3.3, please open a separate PR. Your call. @mjivraja

**Comment by mjivraja** [comment]
> Cherry-picks: ✔️ [24f50d2](https://github.com/theforeman/foreman-documentation/commit/24f50d2497f51b949a1409f918a36bc5ae47901a)..[584dfd1](https://github.com/theforeman/foreman-documentation/commit/584dfd1a6b569ab753ca67f65d7ecd6f0944443b) 3.9 -> 3.9 [0d87b85](https://github.com/theforeman/foreman-documentation/commit/0d87b850aadfe895702ee2ac10601f7c947cd45d)..[3afd09c](https://github.com/theforeman/foreman-documentation/commit/3afd09c653ebe85b4b685373cb25f0b3e645ca47) 3.8 -> 3.8 [f70658b](https://github.com/theforeman/foreman-documentation/commit/f70658bc920eae72818d5853920b4c29b0d1ce0d)..[0bc5c22](https://github.com/theforeman/foreman-documentation/commit/0bc5c2201c3dd761097c342aba303c35e0eb732f) 3.7 -> 3.7 [0e1cfa4](https://github.com/theforeman/foreman-documentation/commit/0e1cfa44c6fcda198cc3e2c4258ab325e345c981)..[244dbdc](https://github.com/theforeman/foreman-documentation/commit/244dbdcc128b61d2b2a438669dcbea9091ea7c4f) 3.6 -> 3.6 [eeafa28](https://github.com/theforeman/foreman-documentation/commit/eeafa284e7d0b5af04b068e126d069e305d98e78)..[56da30c](https://github.com/theforeman/foreman-documentation/commit/56da30caafbed9940e252a5164baa4a505c14b41) 3.5 -> 3.5 [e3c9721](https://github.com/theforeman/foreman-documentation/commit/e3c9721f2453238639157aedd2204b45251a8b05)..[4cc5804](https://github.com/theforeman/foreman-documentation/commit/4cc580406baad38719973f9b5226a07bdc738945) 3.4 -> 3.4
> 
> There was a merge conflict on 3.3. If you want to backport to 3.3, please open a separate PR. Your call. @mjivraja

I find it is required even further. Here are the PRs resolving the conflict:

3.1: https://github.com/theforeman/foreman-documentation/pull/2649
3.2: https://github.com/theforeman/foreman-documentation/pull/2650
3.3: https://github.com/theforeman/foreman-documentation/pull/2651

**Pull request merged**
